### DISCUSSION TITLE
How to update the boilerplate components and utils and hook etc to the latest versions

### DISCUSSION THREAD

**Discussion by johnernest02-EYGDS** [infinitered/ignite#2865] [Q&A]
I have started using Ignite on my project 3 months ago, and the theming support was released 2 months ago ([#2336](https://github.com/infinitered/ignite/pull/2636/files)). How do I get the latest changes into my project? My ignite-cli is already at v10.0.5.

**Comment by frankcalise** [comment]
hey @johnernest02-EYGDS - basically there is no push a button upgrade path. You can follow along the PRs, [Ignite Cookbook recipes](https://ignitecookbook.com) or leverage the `ignite-diff-purge` mentioned in the [docs](https://docs.infinite.red/ignite-cli/concept/Upgrades/).

You also have the ability to spin up a new Ignite project, copy over the components directory, theme directory, etc if that seems like a quicker path - but really depends on your project and how much customization you've done since then.

Having ignite-cli globally installed has no effect on anything and generally I would say don't install it, just use it via `npx ignite-cli@latest`.

Hope this helps
